ctapipe/coordinates/ground_frames.py

```diff
@@ -218,3 +120,81 @@ def project_to_ground(tilt_system):
     y_projected = y_initial - trans[2][1] * z_initial / trans[2][2]
 
     return GroundFrame(x=x_projected * unit, y=y_projected * unit, z=0 * unit)
+
+
+class GroundCoordinate(BaseCoordinate):
+    """
+
+    """
+    system_order = np.array(["GroundFrame", "TiltedGroundFrame"])
+
+    transformations = np.array([ground_to_tilted])
+    reverse_transformations = np.array([project_to_ground])
+
+    def __init__(self, pointing_direction=None):
+        """
+        Parameters
+        ----------
+        focal_length: ndarray
+            Focal length of telescope
+        telescope_pointing: HorizonFrame
+            Pointing direction of telescope
+        array_pointing: HorizonFrame
+            Pointing direction of array
+        rotation: ndarray
+            Rotation angle of camera in telescope
+        """
+        self.pointing_direction = pointing_direction
+
+        prop_dict = dict()
+        for key in self.__dict__:
+            prop_dict[key] = self.__dict__[key]
+        self.properties = prop_dict
+
+        return
+
+    def copy_properties(self):
+        """
+        Create a copy of the shared class parameters to share with other classes
+
+        Returns
+        -------
+        dict: Dictionary of shared class parameters
+        """
+        properties = self.properties
+        return properties
```

Comment:
This function seems to do nothing.  Why not just access the properties directly? If you want a copy, call copy explicitly? (e.g. pass `copy(ground_coord.properties)` above)

Reply:
Yeah this is a horrid bit of code and I thought someone might pull me up on it :-). This is there in case we want to copy across the properties of the base class while ignoring the the properties of the inherited class. There may be a nice pythonic way of doing this, I just don't know it.

Reply:
But it does not copy